File: C/Assignment14/Program1.c
///////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// File name :   Program1.c
// Description : Program which accept number of rows & number of columns from user & display below pattern
// Author :      Pallavi Omprakash Malewar
// Date :        11/07/2025
//
//////////////////////////////////////////////////////////////////////////////////////////////////////////////


/*
Input :     iRow = 4      iCol = 4

Output :    1   2   3   4             
            5   6   7   8    
            9   1   2   3       
            4   5   6   7     
*/


#include<stdio.h>

void Pattern(int iRow , int iCol)
{
    int i = 0 , j = 0 ,iCnt = 1 ;

    for(i = 1 ; i <= iRow ; i++)
    {
        for(j = 1 ; j <= iCol ; j++)
        {
            if(iCnt <= 9)
            {
                printf("%d\t" , iCnt);
                iCnt++;
            }
            else
            {
                iCnt = 1;

                printf("%d\t" , iCnt);
                iCnt++;
            }
        }
        printf("\n");
    }
}

int main()
{
    int iValue1 = 0 , iValue2 = 0;

    printf("Enter Number of Rows & Columns : ");
    scanf("%d %d" ,&iValue1 , &iValue2);

    Pattern(iValue1 , iValue2);

    return 0;

}

File: JAVA/Assignment 14/Program1.java
/////////////////////////////////////////////////////////////////////////////////////////
//                                                                                     //
// Problem Statement : Write a Java Program which adds two matrices.                   //
// Author : Pallavi Omprakash Malewar                                                  //
// Date : 6 Aug 2026                                                                   //
//                                                                                     //
/////////////////////////////////////////////////////////////////////////////////////////


import java.util.Scanner;

class Program1
{
    public static void main(String A[])
    {
        int row, col;

        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter Number of Rows : ");
        row = sobj.nextInt();

        System.out.print("Enter Number of Columns : ");
        col = sobj.nextInt();


        int a[][] = new int[row][col];

        System.out.println("Enter elements of Matrix A : ");

        for(int i = 0; i < row; i++)
        {
            for(int j = 0; j < col; j++)
            {
                System.out.print("Element [" + (i + 1) + "," + (j + 1) + "] : ");
                a[i][j] = sobj.nextInt();
            }
        }


        int b[][] = new int[row][col];

        System.out.println("Enter elements of Matrix B : ");

        for(int i = 0; i < row; i++)
        {
            for(int j = 0; j < col; j++)
            {
                System.out.print("Element [" + (i + 1) + "," + (j + 1) + "] : ");
                b[i][j] = sobj.nextInt();
            }
        }


        int c[][] = new int[row][col];


        for(int i = 0; i < row; i++)
        {
            for(int j = 0; j < col; j++)
            {
                c[i][j] = a[i][j] + b[i][j];
            }
        }


        System.out.println("\nMatrix A : ");

        for(int i = 0; i < row; i++)
        {
            for(int j = 0; j < col; j++)
            {
                System.out.print(a[i][j] + "\t");
            }
            System.out.println();
        }


        System.out.println("\nMatrix B : ");

        for(int i = 0; i < row; i++)
        {
            for(int j = 0; j < col; j++)
            {
                System.out.print(b[i][j] + "\t");
            }
            System.out.println();
        }


        System.out.println("\nAddition of Matrix A and Matrix B : ");

        for(int i = 0; i < row; i++)
        {
            for(int j = 0; j < col; j++)
            {
                System.out.print(c[i][j] + "\t");
            }
            System.out.println();
        }

        sobj.close();
    }
}